crowbar_framework/lib/crowbar/backup/restore.rb

```diff
@@ -23,26 +23,106 @@ def initialize(backup)
         @backup = backup
         @data = @backup.data
         @version = @backup.version
+        @status = {}
       end
 
       def restore
-        actions = [
+        self.class.restore_steps_path.delete if self.class.restore_steps_path.exist?
+
+        thread = Thread.new do
```

Comment:
Lint/UselessAssignment: Useless assignment to variable - `thread`. (https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)